### PR TITLE
fix: flutter authentication blogpost

### PR DESCRIPTION
**Pull request by vinckr**

### CODE REVIEW

**Comment by vinckr** [comment]
@lucaIaconelli 

Here is your blogpost 🎉 
Feel free to review it before we merge.

**Comment by LucaIaconelli** [comment]
Hey @vinckr 

Thanks it looks very cool.

Maybe if there was also the possibility to include it in the features for a few days it would be fantastic.

Thank you and I hope that the collaboration between IGLU and Ory will continue over time. (If you need Flutter resources, we have the best).

**Comment by vinckr** [comment]
For sure, I always forget we have this new "featured" feature now 😁 .

**Comment by vinckr** [comment]
@zeropaper 
@BlueAcidFrog 
This is done from our side, can you please review and merge 🙏 
Ping me if anything should be changed.

File: src/markdown/blog/login-flutter-authentication-example-api-open-source.mdx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+teaser: |
+  Add login, registration, user and profile management to Flutter web applications in minutes using the Ory Kratos open source project. Includes code examples for Dart!
+
+seo:
+  title: |
+    Authentication for Flutter Web Application
+  description: |
+    Add open source login to any Flutter Web App using free open source.
+  keywords: |
+    login, auth, authentication, registration, flutter, dart, mobile, ios, android, mobile authentication, open source, ory, ory kratos
```
Comment (BlueAcidFrog) [inline src/markdown/blog/login-flutter-authentication-example-api-open-source.mdx:16]:
```suggestion
    login, password recovery, account verification, auth, authentication, registration, flutter, dart, mobile, ios, android, mobile authentication, open source, ory, ory kratos
```

**Review by BlueAcidFrog — CHANGES_REQUESTED** [review]
A couple of relevant keywords to be added please

**Comment by vinckr** [comment]
thanks,
done.

**Review by BlueAcidFrog — APPROVED** [review]

**Review by zeropaper — APPROVED** [review]
@BlueAcidFrog It looks OK to me too.

Should I merge? (BTW, Do you have the access rights to merge?)

**Comment by vinckr** [comment]
@zeropaper 
@BlueAcidFrog 
please merge 🙏

**Pull request merged**